change/@microsoft-fast-components-ad8eb6c8-9106-4890-ad00-49f44659b24a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "add function to support programmatic focus for firefox and added test",
```

Comment:
I think this could be more clear - this is only necessary where delegatesFocus isn't supported which is technically both Firefox and iOS safari. We don't need to call out both by name, but I think explaining that this relates to mitigating the lack of delegatesFocus would help.

Reply:
updated